tests/acceptance/services/lazy-loader-test.js

```diff
@@ -59,3 +59,89 @@ test('_getRouteNamesFromUrl', function(assert) {
     assert.notOk(service._getRouteNameFromUrl('/doesntmatch'));
   });
 });
+
+test('dependsOn doesnt break if nothing is specified', function(assert) {
+  const service = getSubject();
+  service.setBundles([{
+    name: 'bundleA',
+    urls: ['stuff.js'],
+    // dependsOn: [],
```

Comment:
I like that you left it commented out on purpose to make it explicit that dependsOn is undefined.